src/datasets/utils/resources/tasks.json

```diff
@@ -77,10 +77,23 @@
             "other"
         ]
     },
+    "automatic-speech-recognition": {
+        "description": "recognising speech within audio and converting it to text",
+        "options": [
+            "speech-recognition",
+            "visual-speech-recognition",
+            "distant-speech-recognition",
+            "robust-speech-recognition",
+            "seq2seq-speech-recognition",
```

Comment:
`seq2seq-speech-recognition` is just regular `speech-recognition` with seq2seq models ?
Not sure this should be a separate task, since the difference is about an aspect of the model. I think we can remove it

Reply:
yeah good idea - i'll remove it!